recipes/sage/meta.yaml

```diff
@@ -0,0 +1,193 @@
+{% set name = "sage" %}
+{% set version = "7.5.1" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+  skip: true  # [py!=27]
+  features:
+    - blas_openblas
+
+requirements:
+  build:
+    - blas 1.1 openblas
+    - numpy 1.11.*
+    - openblas 0.2.19|0.2.19.*
+    - python
+    - r-base
+  run:
+    - alabaster
+    - appnope  # [osx]
+    - arb
+    - babel
+    - backports.shutil_get_terminal_size
+    - backports_abc
+    - bdw-gc
+    - blas 1.1 openblas
+    - boost-cpp 1.63.*
+    - brial
+    - bzip2 1.0.*
+    - cddlib
+    - certifi
+    - cliquer
+    - configparser
+    - cvxopt
+    - cycler
+    - cysignals
+    - decorator
+    - docutils
+    - ecl
+    - eclib
+    - ecm
+    - entrypoints
+    - fflas-ffpack
+    - flask
+    - flask-babel
+    - flask-openid
+    - flintqs
+    - fplll
+    - fpylll
+    - freetype 2.7|2.7.*
+    - functools32
+    - future
+    - gap
+    #- gcc
+    - gf2x
+    - gfan
+    - giac
+    # is this necessary?
+    #- git
+    - givaro
+    - glpk
+    - gmp >=5.0.1,<7
+    - gsl
+    - imagesize
+    - iml
+    - ipykernel
+    - ipython
+    - ipython_genutils
+    - ipywidgets
+    - itsdangerous
+    - jinja2
+    - jsonschema
+    - jupyter_client
+    - jupyter_core
+    - lcalc
+    - libflint
+    - libgap
+    - libgd
+    - libiconv
+    - libpng >=1.6.28,<1.7
+    - linbox
+    - lrcalc
+    - m4ri
+    - m4rie
+    - markupsafe
+    - matplotlib
+    #- maxima
+    - mistune
+    - mpc
+    - mpfi
+    - mpfr 3.*
+    - mpmath
+    - nauty
+    - nbconvert
+    - nbformat
+    - ncurses 5.9*
+    - networkx
+    - notebook
+    - ntl
+    - numpy 1.11.*
+    - openblas 0.2.19|0.2.19.*
+    - palp
+    - pari
+    - path.py
+    - pathlib2
+    - pexpect
+    - pickleshare
+    - pillow
+    - planarity
+    - ppl
+    - prompt_toolkit
+    - psutil
+    - ptyprocess
+    - pycrypto
+    - pygments
+    - pynac
+    - pyparsing
+    - python
```

Comment:
Are runtime dependencies automatically build time dependencies? If so, then this is a duplicate. Also, this must be python 2.7. Not sure if that's automatically the case here already.